server/app.ts

```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import router from './routes'
+
+const app = express();
+app.set('port', process.env.PORT || 3500);
+dotenv.config();
+
+app.use(express.json());
```

Comment:
`app.use()` takes a series of middlewares as an array, 
so I prefer to use it in that way
```js
app.use([cookieParser(), compression(), ..etc]);
```